tests/backend/test_speculos.py

```diff
@@ -172,6 +173,22 @@ def test_navigate_and_compare_until_snap(self):
                         ROOT_SCREENSHOT_PATH, "generic", 0, 2)
                 self.assertTrue(ret)
 
+    def test_navigate_and_compare(self):
```

Comment:
Not necessary to test non-nominal cases ?

Reply:
tests added :)